php/php.editor/src/org/netbeans/modules/php/editor/parser/api/Utils.java

```diff
@@ -78,16 +79,24 @@ public static Comment getCommentForNode(Program root, ASTNode node) {
                 int start = possible.getEndOffset() + 1;
                 int end = getNodeRangeLocatorEndOffset(node);
                 if (start <= end) {
+                    //Check that a possible comment and a node are in the same block
+                    ASTNode cnode = getNodeAtOffset(root, start);
```

Comment:
Does `c` mean "comment" or "check"?
I prefer another name to `cnode`.

Reply:
Renamed to `nodeAtStartOffset`.

Reply:
Nice.